toolchain/check/context.h

```diff
@@ -351,6 +342,11 @@ class Context {
   // FinalizeImportRefBlock() will produce an inst block for them.
   llvm::SmallVector<SemIR::InstId> import_ref_ids_;
 
+  // Map from an AnyBindingPattern inst to precomputed parts of the
+  // pattern-match SemIR for it.
+  //
+  // TODO: Consider putting this behind a narrower API to guard against emitting
+  // multiple times.
```

Comment:
I think this TODO comment was referring to the accessor function exposing a wide API to the map, not to the field itself, and would be better kept with the accessor function.

Reply:
Done